README.md

```diff
@@ -17,17 +17,62 @@ Then run the 'sbom' task on a project:
     ===> Verifying dependencies...
     ===> CycloneDX SBoM written to bom.xml
 
+Configuration
+-------------
+
+You can configure additional SBoM metadata in your `rebar.config`:
+
+    {rebar3_sbom, [
+        {sbom_manufacturer, #{          % Optional, all fields inside are optional
+            name => "Your Organization",
+            url => "https://example.com",
```

Comment:
Can I also put in `url => ["https://example.com", "https://another-example.com"]`? CycloneDX spec says the URL should allow multiple URLs and the type is `array of string`.